### ISSUE TITLE
No Audit with Opera 26 Stable

### ISSUE THREAD

**Issue by ghost** [GoogleChrome/accessibility-developer-tools-extension#26]
Does not show up in Audits with Opera 26.0.1656.24 Stable and Opera 27.0.1689.2 DEV (windows).
1. Install [Download Chrome Extension](https://addons.opera.com/en/extensions/details/download-chrome-extension-9/) for downloading from Chrome Web Store and activate extension
2. Install [Accessibility Developer Tools](https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb) and activate extension
3. Hit Ctrl Shift I to start _Developer Tools_
4. Select _Audits_ in Developer Tools
5. Missing point _Accessibility_ to select

No error messages found in Console of Developer Tools in Opera.

Extension works with Iron 38.0.2050.0 (295000).

Seems, Extension does not work in Chromium clone Opera 26.
